nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/StandardFlowService.java

```diff
@@ -173,7 +175,8 @@ public static StandardFlowService createClusteredInstance(
             final RevisionManager revisionManager,
             final Authorizer authorizer) throws IOException {
 
-        return new StandardFlowService(controller, nifiProperties, senderListener, encryptor, true, coordinator, revisionManager, authorizer);
+        return new StandardFlowService(controller, nifiProperties, senderListener, encryptor, true, coordinator, revisionManager, authorizer,
+                FlowSerializationStrategy.WRITE_XML_AND_JSON);
```

Comment:
Due to simmerty and having the same responsibilities for standalone and clustered methods, I would add an argument for this as well. As I found, it is callled only in the `StandardFlowServiceFactoryBean`, and the used enum might be specified there (like in case of standalone)

Reply:
Only NiFi itself runs in clustered mode, the reason we have an extra argument in the standalone instance is for MiNiFi, which needs WRITE_XML_ONLY and calls getStandaloneInstance() directly (not via Spring)